sympy/combinatorics/fp_groups.py

```diff
@@ -516,9 +516,42 @@ def elements(self):
 
 def subgroup_quotient(G, H, fp_group=None):
```

Comment:
I think it should be called `parent_group`, rather than `fp_group` - it's clearer that way. And the docstring should explain what that arguments is for